launch.py

```diff
@@ -7,7 +7,7 @@
 
 
 def mod_param(name, mods):
-    return ' -{}="{}" '.format(name, ";".join(mods))
+    return ' -{}="{}" '.format(name, ";".join(mods)).replace('/arma3/', '')
```

Comment:
needs more cowbell

Reply:
I believe there is sufficient amount of cowbell